compiler/src/iree/compiler/Codegen/LLVMCPU/KernelDispatch.cpp

```diff
@@ -1240,14 +1240,14 @@ static LogicalResult setRootConfig(
   return success();
 }
 
-/// Sets the lowering configuration for linalg.conv_2d_nhwc_hwcf and
-/// linalg.depthwise_conv_2d_nhwc_hwc operations.
+/// Sets the lowering configuration for linalg.conv_2d_nhwc_hwcf,
+/// linalg.conv_2d_nchw_fchw, and linalg.depthwise_conv_2d_nhwc_hwc operations.
```

Comment:
Let's update the comment to `... for conv ops. See below for supported conv ops`. Then we won't have to update it every time. The snippets are close enough, so I think it's fine.